sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -1086,3 +1089,56 @@ def _more_tags(self):
         all_tests = list(_yield_all_checks(Estimator()))
         assert check_methods_sample_order_invariance not in all_tests
         assert check_methods_subset_invariance not in all_tests
+
+
+def test_check_outlier_contamination():
+    """Check the test for the contamination parameter in the outlier detectors."""
+
+    # Without any parameter constraints, the estimator will early exit the test by
+    # returning None.
+    class OutlierDetectorWithoutConstraint(OutlierMixin, BaseEstimator):
+        """Outlier detector without parameter validation."""
+
+        def __init__(self, contamination=0.1):
+            self.contamination = contamination
+
+        def fit(self, X, y=None, sample_weight=None):
+            return self
```

Comment:
Codecov is not happy here. I know that an `OutlierDetector` requires `fit` and `predict`, but I am okay with removing it. Alternative is to do `# pragma: no cover`.

I am okay with either option.